spex/extensions/spex-gates/commands/speckit.spex-gates.review-code.md

```
@@ -255,6 +255,81 @@ Compliance % = (Compliant Requirements / Total Requirements) x 100
+
+#### Checking Code for Constants
+
+For each parsed constant:
+
+1. **Search the codebase** for definitions matching the constant name:
+   ```bash
+   rg -n "\b${NAME}\b" --type-not md --type-not yaml -- . | grep -v '^\./specs/' | grep -v '^\./\.specify/'
+   ```
+
+2. **Check value match**: Compare the spec-declared value with the code-defined value. Values are compared as trimmed strings. Report a drift finding if they differ:
+   ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)